AudioStream.js

```diff
@@ -57,7 +57,16 @@ module.exports = class AudioStream {
         return exec(`echo '{ "command": ["cycle", "pause"] }' | socat - /tmp/mpvsocket${this.current}`)
     }
 
-
+    async addToQueue(url) {
+        try {
+            const info = await ytdl.getInfo(url);
+            const title = info.videoDetails.title;
+            const mediaItem = { title, url };
```

Comment:
can we also include the thumbnail url? if ytdl returns it for us

Reply:
Done